[
    {
        "user": "UFTBRB3F1",
        "type": "message",
        "ts": "1600024677.030200",
        "client_msg_id": "90b7bff4-38bd-45a2-a23f-172e24ca59d5",
        "text": "By chance does anyone have the race result (in the format from the racecard end point) for the 18:45 from Musselburgh yesterday ('1.172950042').\n\nFor some reason I can get every other race, but that one throws up an error (I'm guessing Betfair did something at their end) and purely for OCD purposes it would be handy to have it if anyone has it to hand?\n\n```Traceback (most recent call last):\n  File \"D:\/Python37\/webpages\/bingo.py\", line 41, in &lt;module&gt;\n    race_cards = trading.race_card.get_race_result(market_ids=['1.172950042'])\n  File \"D:\\Python38\\lib\\site-packages\\betfairlightweight\\endpoints\\racecard.py\", line 94, in get_race_result\n    (response, response_json, elapsed_time) = self.request(\n  File \"D:\\Python38\\lib\\site-packages\\betfairlightweight\\endpoints\\racecard.py\", line 113, in request\n    check_status_code(response)\n  File \"D:\\Python38\\lib\\site-packages\\betfairlightweight\\utils.py\", line 31, in check_status_code\n    raise StatusCodeError(response.status_code)\nbetfairlightweight.exceptions.StatusCodeError: Status code error: 500```",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g771e110be94",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a771e110be945b86bede9d5c1bb551e3.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png",
            "first_name": "",
            "real_name": "Newbie99",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "vjmehra",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "7PLQ",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "By chance does anyone have the race result (in the format from the racecard end point) for the 18:45 from Musselburgh yesterday ('1.172950042').\n\nFor some reason I can get every other race, but that one throws up an error (I'm guessing Betfair did something at their end) and purely for OCD purposes it would be handy to have it if anyone has it to hand?\n\n"
                            }
                        ]
                    },
                    {
                        "type": "rich_text_preformatted",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Traceback (most recent call last):\n  File \"D:\/Python37\/webpages\/bingo.py\", line 41, in <module>\n    race_cards = trading.race_card.get_race_result(market_ids=['1.172950042'])\n  File \"D:\\Python38\\lib\\site-packages\\betfairlightweight\\endpoints\\racecard.py\", line 94, in get_race_result\n    (response, response_json, elapsed_time) = self.request(\n  File \"D:\\Python38\\lib\\site-packages\\betfairlightweight\\endpoints\\racecard.py\", line 113, in request\n    check_status_code(response)\n  File \"D:\\Python38\\lib\\site-packages\\betfairlightweight\\utils.py\", line 31, in check_status_code\n    raise StatusCodeError(response.status_code)\nbetfairlightweight.exceptions.StatusCodeError: Status code error: 500"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "subtype": "channel_join",
        "user": "U01A5GVTBST",
        "text": "<@U01A5GVTBST> has joined the channel",
        "type": "message",
        "ts": "1600029037.030400"
    },
    {
        "subtype": "channel_join",
        "user": "U01AH6Q3GUV",
        "text": "<@U01AH6Q3GUV> has joined the channel",
        "type": "message",
        "ts": "1600029516.030600"
    },
    {
        "user": "U9JHLMZB4",
        "type": "message",
        "ts": "1600063281.033000",
        "client_msg_id": "3eaded50-8404-4a4b-86ab-2b2b26758ac9",
        "text": "<@UFTBRB3F1> I think you're going to be out of luck. This for the 18:15 works, `race_cards = trading.race_card.get_race_result(market_ids=['1.172974372'])` but stick '1.172950042' in there instead and it's still throwing an http 500 (server error) response. So def. a problem at the Betfair end and one likely to affect everybody.",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g951ddcb43e7",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/3951ddcb43e788a387d6daf330dad5ca.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0018-72.png",
            "first_name": "",
            "real_name": "Peter Coles",
            "display_name": "Peter",
            "team": "T4G9NBD2M",
            "name": "peter",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "hY0",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "user",
                                "user_id": "UFTBRB3F1"
                            },
                            {
                                "type": "text",
                                "text": " I think you're going to be out of luck. This for the 18:15 works, "
                            },
                            {
                                "type": "text",
                                "text": "race_cards = trading.race_card.get_race_result(market_ids=['1.172974372'])",
                                "style": {
                                    "code": true
                                }
                            },
                            {
                                "type": "text",
                                "text": " but stick '1.172950042' in there instead and it's still throwing an http 500 (server error) response. So def. a problem at the Betfair end and one likely to affect everybody."
                            }
                        ]
                    }
                ]
            }
        ],
        "reactions": [
            {
                "name": "+1",
                "users": [
                    "UBS7QANF3"
                ],
                "count": 1
            }
        ]
    },
    {
        "user": "U9JHLMZB4",
        "type": "message",
        "ts": "1600063334.033900",
        "client_msg_id": "d33301a6-8688-4e15-a991-4075e10e14a9",
        "text": "Also a warning to anybody collecting results in bulk. Including '1.172950042' in a list will cause the entire request to fail.",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g951ddcb43e7",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/3951ddcb43e788a387d6daf330dad5ca.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0018-72.png",
            "first_name": "",
            "real_name": "Peter Coles",
            "display_name": "Peter",
            "team": "T4G9NBD2M",
            "name": "peter",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "NXbA5",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Also a warning to anybody collecting results in bulk. Including '1.172950042' in a list will cause the entire request to fail."
                            }
                        ]
                    }
                ]
            }
        ],
        "reactions": [
            {
                "name": "+1",
                "users": [
                    "U0198BFR9GR"
                ],
                "count": 1
            }
        ]
    },
    {
        "user": "UBS7QANF3",
        "type": "message",
        "ts": "1600064906.034700",
        "client_msg_id": "a7d17f85-c727-413c-a4ff-07b3b7c4fc90",
        "text": "Thanks for the warning, going to have to make a code change to account for that",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "gaaf844a4a90",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/eaaf844a4a905431d83430e563b077aa.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0011-72.png",
            "first_name": "",
            "real_name": "Maurice Berk",
            "display_name": "Mo",
            "team": "T4G9NBD2M",
            "name": "maurice",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "9UrH",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Thanks for the warning, going to have to make a code change to account for that"
                            }
                        ]
                    }
                ]
            }
        ]
    }
]